primitives/traits/src/lib.rs

```diff
@@ -110,3 +110,11 @@ pub trait RemoveInvulnerables<AccountId> {
         num_invulnerables: usize,
     ) -> Vec<AccountId>;
 }
+
+/// Helper trait for pallet_collator_assignment to be able to not assign collators to container chains with no credits
+/// in pallet_services_payment
+pub trait RemoveParaIdsWithNoCredits {
+    /// Remove para ids with not enough credits. The resulting order will affect priority: the first para id in the list
+    /// will be the first one to get collators.
```

Comment:
First meaning the paraId that has higher credits?

Reply:
Could be, now the input order is random and the output order is also random (same as input). I made this trait thinking about the case you mention, so in the future we can reorder the para ids and sort them by number of credits or by priority fee for instance.

Reply:
good!